Remove unused component reference

**Describe the bug** A clear and concise description of what the bug is.

I was running the linter and noticed a warning about an unused import in `src/routes/+page.svelte`. Remove the `import Coin from '../lib/Coin.svelte';` import. 

```diff
<script>
	import Board from '../lib/Board.svelte';
-	import Coin from '../lib/Coin.svelte';
</script>

<header>
	<h1>The beginnings of the classic Connect Four game.</h1>
</header>
<main>
	<h2>Welcome to SvelteKit</h2>
	<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
	<Board />
</main>
Footer © 2022 GitHub, Inc. Footer navigation

<style>
	h1 {
		padding: 1rem;
		border: 2rem dotted blue;
	}
</style>

```

**To Reproduce** Steps to reproduce the behavior:

1. Run `npm run lint`
2. Notice the warning in the console.

```
/Users/taco/dev/oss/connect-four/src/routes/+page.svelte
  3:9  warning  'Coin' is defined but never used  @typescript-eslint/no-unused-vars
```

**Expected behavior** A clear and concise description of what you expected to
happen.

No warning about an unused import

**Screenshots** If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**

- OS: [e.g. iOS]
- Browser [e.g. chrome, safari]
- Version [e.g. 22]

**Smartphone (please complete the following information):**

- Device: [e.g. iPhone6]
- OS: [e.g. iOS8.1]
- Browser [e.g. stock browser, safari]
- Version [e.g. 22]

**Additional context** Add any other context about the problem here.

--- a/src/routes/+page.svelte
+++ b/src/routes/+page.svelte
@@ -1,6 +1,5 @@
 <script>
 	import Board from '../lib/Board.svelte';
-	import Coin from '../lib/Coin.svelte';
 </script>
 
 <header>

